cmake/qtifw/install_unix_createsymlinks.qs

```diff
@@ -47,10 +47,30 @@ function Component()
       component.addElevatedOperation("Execute", "ln", "-sf", "@TargetDir@/PreProcess/GrndTempCalc/SlabGHT.idd", linktarget,
                                      "UNDOEXECUTE", "rm", linktarget + "/SlabGHT.idd");
 
+      if( kernel == "linux") {
+        // Historical: TODO: is this needed really?
+        component.addElevatedOperation("Execute", "ln", "-sf", "@TargetDir@/energyplus", linktarget + "/EnergyPlus",
```

Comment:
At one point, users complained about the casing of the energyplus binary, so we symlinked it to get it right on case sensitive platforms (Linux).  Whether this symlinking occurs at build time and the symlink is just packaged up, or if it occurs here at install time, I don't have a preference.